web/twa-vis-framework/library/src/ts/ui/scenario_handler.ts

```diff
@@ -74,43 +74,40 @@ class ScenarioHandler {
      */
     public showSelector() {
         let container = document.getElementById("scenario-container");
-
-        if(container == null) {
+        
+        if (container == null) {
             // Create and add the UI element
             container = document.createElement("div");
             container.id = "scenario-container";
-
+            
             container.innerHTML = `
                 <div id="scenario-blocker"></div>
                 <div id="scenario-popup">
-                    <div id="scenario-close"><i class="fa fa-times"></i></div>
-                    <div id="scenario-header">
-                        <b style="font-size: 120%;">Select a scenario:</b><br/><br/>
-                        <p>Please identify a scenario from the list below, then select the 'View' button to plot its data.</p>
+                <button id="scenario-close"><i class="fa fa-times"></i></button>
+                <div id="scenario-header">
+                <b style="font-size: 120%;">Select a scenario:</b><br/><br/>
+                <p>Please identify a scenario from the list below, then select the 'View' button to plot its data.</p>
                     </div>
                     <div id="scenario-inner">
                         <div id="scenario-loading" style="display: table;">
                             <img style="width: 200px;" src="loading.gif"></img>
                             <p style="color: grey; font-style: italic;">Loading scenario details, please wait...</p>
-                        </div>
-                    </div>
-                </div>
-            `;
-            document.body.appendChild(container);
-
-            // Send of request to get the scenario details
-            let self = this;
+                            </div>
+                            </div>
+                            </div>
+                            `;
+                            document.body.appendChild(container);
+                            
+                            // Send of request to get the scenario details
+                            let self = this;
```

Comment:
Has the formatting gone wrong here?

Reply:
Yes looks like autoformatter doesn't work on string injected HTML in the .js files. Fixed now!